hasura/migrations/default/1674972789147_alter_table_public_circles_add_column_guild_role_id/up.sql

```diff
@@ -0,0 +1,2 @@
+alter table "public"."circles" add column "guild_role_id" integer
```

Comment:
I didn't do this because this looked a lot more loosey goosey and i wanted to make this first class. I think this table is good for 3p integrations run by others. We can work together on how to unify/improve or set clean boundaries around this.